src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -560,7 +568,7 @@ private void AddFunctionDescriptors(IEnumerable<FunctionMetadata> functionMetada
                 var workerConfig = _languageWorkerOptions.Value.WorkerConfigs?.FirstOrDefault(c => c.Description.Language.Equals(_workerRuntime, StringComparison.OrdinalIgnoreCase));
 
                 // If there's no worker config, use the default (for legacy behavior; mostly for tests).
-                TimeSpan initializationTimeout = workerConfig?.CountOptions?.InitializationTimeout ?? WorkerProcessCountOptions.DefaultInitializationTimeout;
+                var initializationTimeout = workerConfig?.CountOptions?.InitializationTimeout ?? WorkerProcessCountOptions.DefaultInitializationTimeout;
```

Comment:
Just curious - why was this type changed?

Reply:
Kept it consistent with line above, where `workerConfig` var is used instead of `ClassName`.

Reply:
I think using Timespan makes more sense here. With `workerConfig`, we know the type of the variable, but with `initializationTimeout` it's not clear through assignment what the type is and so it makes sense to be clear with that.